plugins/catalog-backend/src/processing/DefaultCatalogProcessingOrchestrator.ts

```diff
@@ -55,6 +56,13 @@ import {
 } from './util';
 import { CatalogRulesEnforcer } from '../ingestion/CatalogRules';
 import { ProcessorCacheManager } from './ProcessorCacheManager';
+import {
+  addEntityAttributes,
+  TRACER_ID,
+  withActiveSpan,
+} from '../util/opentelemetry';
+
+const tracer = trace.getTracer(TRACER_ID);
```

Comment:
Same here, should this be moved into the constructor or even the individual step methods?